res/sectors.xsd

```diff
@@ -39,7 +39,7 @@
               <xs:attribute name="Publisher" type="xs:string" use="required" />
               <xs:attribute name="Title" type="xs:string" use="required" />
               <xs:attribute name="Author" type="xs:string" use="optional" />
-              <xs:attribute name="Ref" type="xs:anyURI" use="required" />
+              <xs:attribute name="Ref" type="xs:anyURI" use="optional" />
```

Comment:
This change is based on the fact the xml returned by the metadata API for Reaver's Deep (M1105) doesn't have the Ref element (possibly other sectors as well)